src/v2/view-builder/views/phone/ChallengeAuthenticatorPhoneView.js

```diff
@@ -0,0 +1,244 @@
+            'type': 'button'
+          },
+          className: 'button button-primary phone-authenticator-verify__button--primary',
+          title: primaryButtonTitle,
+          click () {
+            // Call the API to send a code via primary mode
+            this.options.appState.trigger('invokeAction', 'currentAuthenticatorEnrollment-resend');
+            this.model.set('mode', primaryMode);
+          },
+        })
+      );
+      if (secondaryMode) {
+        const secondaryButtonTitle = (secondaryMode === 'sms')
+          ? loc('oie.phone.sms.secondaryButton', 'login')
+          : loc('oie.phone.call.secondaryButton', 'login');
+        this.add(
+          createButton({
+            attributes: {
+              'data-se': 'phone-authenticator-verify__button--secondary',
+              'type': 'button'
+            },
+            className: 'phone-authenticator-verify__button--secondary',
+            title: secondaryButtonTitle,
+            click () {
+              // Call the API to send a code via secondary mode
+              this.options.appState.trigger('invokeAction', 'currentAuthenticatorEnrollment-resend');
+              this.model.set('mode', secondaryMode);
+            },
+          })
+        );
+      }
+
+      this.listenTo(this.model, 'error', this.startPolling.bind(this));
+      this.listenTo(this.model, 'change:mode', this.render.bind(this));
+    },
+
+    saveForm () {
+      BaseForm.prototype.saveForm.apply(this, arguments);
+      this.stopPolling();
+      // TODO: abort ongoing request. (https://oktainc.atlassian.net/browse/OKTA-244134)
+    },
+
+    /* eslint max-statements: [2, 25]*/
+    render (e, mode) {
+
+      BaseForm.prototype.render.apply(this, arguments);
+
+      const buttonBar = this.el.querySelector('.o-form-button-bar');
+      const inputFieldset = this.el.querySelector('.o-form-fieldset');
+      const subtitleElement = this.el.querySelector('.okta-form-subtitle');
+      const primaryButton = this.el.querySelector('.phone-authenticator-verify__button--primary');
+      const secondaryButton = this.el.querySelector('.phone-authenticator-verify__button--secondary');
+
+      const maskedPhone = this.options.appState.get('authenticatorProfile').phone;
+      if (mode === 'sms' || mode === 'voice') {
+        // SMS or Voice code sent screen..
+        const sendText = (mode === 'sms')
+          ? loc('oie.phone.verify.sms.codeSentText', 'login')
+          : loc('mfa.calling', 'login');
+        const enterCodeText = loc('oie.phone.verify.enterCodeText', 'login');
+  
+        // Override message in form subtitle so that we can add html content to it.
+        // Courage form subtitle doesn't support html tags.
+        subtitleElement.innerText = '';
+        this.add(
+          `${sendText}&nbsp;<span class='strong'>${maskedPhone}.</span>&nbsp;${enterCodeText}`,
+          '.okta-form-subtitle'
+        );
+        buttonBar.classList.remove('hide');
```

Comment:
The reason why I'm doing `buttonBar` and not the `submit` button only is because when a single option remediation takes place, button-bar has extra spacing, moreover, we need to avoid any buttons that come as a part of remediation and show our primary and secondary options.